web/app.js

```diff
@@ -493,6 +493,7 @@ const PDFViewerApplication = {
       abortSignal: this._globalAbortController.signal,
       enableHWA,
       supportsPinchToZoom: this.supportsPinchToZoom,
+      enableAutolinking: AppOptions.get("enableAutolinking"),
```

Comment:
Here, and elsewhere throughout the patch.
```suggestion
      enableAutoLinking: AppOptions.get("enableAutoLinking"),
```